lib/yjit-metrics/report_types/bloggable_speed_report.rb

```diff
@@ -491,55 +486,125 @@ def svg_object(benchmarks: @benchmark_names)
 
                     tooltip_text = "#{"%.2f" % speedup}x baseline speed (#{human_name})"
 
-                    bar_left = bars_width_start + config_idx * bar_width
-                    bar_right = bar_left + bar_width
-                    bar_lr_center = bar_left + 0.5 * bar_width
-                    bar_top = plot_effective_top + (1.0 - bar_height_ratio) * plot_effective_height
-
                     if config == @baseline_config
                       next
                     end
 
-                    svg.rect \
-                        x: ratio_to_x(bar_left),
-                        y: ratio_to_y(bar_top),
-                        width: ratio_to_x(bar_width),
-                        height: ratio_to_y(bar_height_ratio * plot_effective_height),
-                        fill: ruby_config_bar_colour[config],
-                        data_tooltip: tooltip_text
-
-                    # Whiskers should be centered around the top of the bar, at a distance of one stddev.
-                    top_whisker_y = bar_top - stddev_ratio * plot_effective_height
-                    svg.line x1: ratio_to_x(bar_left), y1: ratio_to_y(top_whisker_y),
-                        x2: ratio_to_x(bar_right), y2: ratio_to_y(top_whisker_y),
-                        **Theme.stddev_marker_attrs
-                    bottom_whisker_y = bar_top + stddev_ratio * plot_effective_height
-                    svg.line x1: ratio_to_x(bar_left), y1: ratio_to_y(bottom_whisker_y),
-                        x2: ratio_to_x(bar_right), y2: ratio_to_y(bottom_whisker_y),
-                        **Theme.stddev_marker_attrs
-                    svg.line x1: ratio_to_x(bar_lr_center), y1: ratio_to_y(top_whisker_y),
-                        x2: ratio_to_x(bar_lr_center), y2: ratio_to_y(bottom_whisker_y),
-                        **Theme.stddev_marker_attrs
+                    bar_data.last[:bars] << {
+                      value: bar_height_ratio,
+                      fill: ruby_config_bar_colour[config],
+                      tooltip: tooltip_text,
+                      stddev_ratio: stddev_ratio,
+                    }
                 end
             end
+        end
 
-            # Below all the bars, we'll want a tick on the bottom axis and a name of the benchmark
-            bars_width_middle = bars_width_start + 0.5 * each_bench_width
-            svg.line x1: ratio_to_x(bars_width_middle), y1: ratio_to_y(plot_bottom_edge),
-                x2: ratio_to_x(bars_width_middle), y2: ratio_to_y(plot_bottom_edge + tick_length),
-                stroke: Theme.axis_color
+        geomeans = ruby_configs.each_with_object({}) do |config, h|
+          next unless @speedup_by_config[config]
+          values = benchmarks.map { |bench| @speedup_by_config[config][ @benchmark_names.index(bench) ]&.first }.compact
+          h[config] = [geomean(values), geo_stddev(values)]
```

Comment:
The other bars use "relative stddev" which is `stddev(values) / mean(values)`.
I don't know if there is an equivalent that makes sense to use here with the geomean.
https://github.com/Shopify/yjit-metrics/blob/6fea005255dbb1a442118037e279fd14efc82bfb/lib/yjit-metrics/report_types/bloggable_speed_report.rb#L130
https://github.com/Shopify/yjit-metrics/blob/6fea005255dbb1a442118037e279fd14efc82bfb/lib/yjit-metrics/report_types/bloggable_speed_report.rb#L489-L490


There is another additional calculation and I'm not sure what that is doing and if we want something similar here.
https://github.com/Shopify/yjit-metrics/blob/6fea005255dbb1a442118037e279fd14efc82bfb/lib/yjit-metrics/report_types/bloggable_speed_report.rb#L148